packages/composite-checkout/demo/index.js

```diff
@@ -157,18 +207,16 @@ const steps = [
 		className: 'checkout__billing-details-step',
 		hasStepNumber: true,
 		titleContent: <ContactFormTitle />,
-		activeStepContent: <ContactForm isComplete={ false } isActive={ true } />,
-		completeStepContent: <ContactForm summary isComplete={ true } isActive={ false } />,
+		activeStepContent: <ContactForm />,
+		completeStepContent: <ContactForm summary />,
```

Comment:
I'm not familiar with this syntax; is

`<ContactForm summary />`

The jsx version of saying

`{ foo }`

instead of 

`{ foo: foo }`?

Reply:
Similar. It's like saying `{ foo: true }`. It's drawn from html, which allows setting boolean attributes without a property, eg: `<button disabled/>`